web/src/components/admin/settings/AdminQueueTab.vue

```diff
@@ -110,7 +110,12 @@ const tasks = computed(() => {
     _tasks.push(...queueInfo.value.waiting_on_deps.map((task) => ({ ...task, status: 'waiting_on_deps' })));
   }
 
-  return _tasks.sort((a, b) => a.id - b.id);
+  return _tasks
+    .map((task) => ({
+      ...task,
+      labels: Object.fromEntries(Object.entries(task.labels).filter(([key]) => key !== 'org-id' && key !== 'repo-id')),
+    }))
+    .sort((a, b) => a.id - b.id);
```

Comment:
`sort` modifies the var.

```suggestion
    .toSorted((a, b) => a.id - b.id);
```